add-to-docs-project/package.json

```diff
@@ -1,22 +1,26 @@
 {
-  "name": "add-to-docs-project",
-  "version": "1.0.0",
-  "description": "Adds all issues labeled `type/docs` to the Docs organization project.",
-  "main": "index.mjs",
-  "repository": "https://github.com/grafana/writers-toolkit/tree/main/add-to-docs-project",
   "author": "@jdbaldry",
+  "dependencies": {
+    "@actions/core": "^1.10.1",
+    "@octokit/graphql": "^8.1.1",
+    "@octokit/rest": "^21.0.2"
+  },
+  "description": "Adds all issues labeled `type/docs` to the Docs organization project.",
+  "devDependencies": {
+    "@types/node": "^22.7.4"
+  },
+  "engines": {
+    "node": ">=22",
+    "npm": ">=10"
+  },
   "license": "Apache-2.0",
+  "main": "index.mjs",
```

Comment:
Should this be `.mts`? https://github.com/grafana/writers-toolkit/blob/main/add-to-docs-project/index.mts

Reply:
Updated to point at `main.mts` after reading about it in https://docs.npmjs.com/cli/v10/configuring-npm/package-json#main